unity-renderer/Assets/UnityGLTF/Scripts/Cache/AssetCache.cs

```diff
@@ -37,8 +36,9 @@ public static RefCountedTextureData AddImage(string uri, string idSuffix, Textur
         /// <param name="texture">The texture to cached image</param>
         public static RefCountedTextureData AddImage(string fullId, Texture2D texture)
         {
-            ImageCacheByUri[fullId] = new RefCountedTextureData(fullId, texture);
-            return ImageCacheByUri[fullId];
+            var refCountedTextureData = new RefCountedTextureData(fullId, texture);
+            ImageCacheByUri[fullId] = refCountedTextureData;
+            return refCountedTextureData;
```

Comment:
This should be equivalent, not requesting a change, just curious if this looks more readable to you

Reply:
the alternative would be 
```
ImageCacheByUri[fullId] = new RefCountedTextureData(fullId, texture);
return ImageCacheByUri[fullId];
```
but I dislike very much the idea of accessing a dictionary element by key twice when it can be avoided :)

Reply:
Well, accessing a dictionary is O(1) but yeah no big deal